main-window.js

```diff
@@ -85,6 +86,17 @@ module.exports = function (config) {
   electron.ipcRenderer.on('goForward', views.goForward)
   electron.ipcRenderer.on('goBack', views.goBack)
 
+  if (mouseForwardBack) {
```

Comment:
I checked the code of `mouse-forward-back`, and at the beginning of the sole JS file it returns without a value if we’re not under Linux (Windows and Mac Electron seems to work fine without this patch.) I assumed (but couldn’t test) in such cases this will be falsy.

Reply:
Thanks, that makes sense to me!